### ISSUE TITLE
I think this page needs to provide a new information about WSL 2

### ISSUE THREAD

**Issue by rkttu** [MicrosoftDocs/Virtualization-Documentation#1305]
Recently, LCOW for moby seems inactive because WSL 2 can handle almost of linux container scenarios (including Kubernetes) on behalf of LCOW. Still, LCOW works in progress, but many developers will choose WSL 2 confidently instead of LCOW natually. So I think this document should introduce and recommend about new WSL 2 docker environment for developers and DevOps engineers.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 87364a28-e1df-eda3-2480-11556ac578be
* Version Independent ID: eebcf9e0-2ff4-d3b5-687f-682adf0b005b
* Content: [Linux Containers on Windows](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/linux-containers#feedback)
* Content Source: [virtualization/windowscontainers/deploy-containers/linux-containers.md](https://github.com/MicrosoftDocs/Virtualization-Documentation/blob/live/virtualization/windowscontainers/deploy-containers/linux-containers.md)
* Service: **windows-containers**
* Product: **windows-containers**
* GitHub Login: @scooley

**Comment by adelcast** [comment]
Yes! the messaging is super confusing. I already spent a non trivial amount of time trying to figure out the different ways I can run containers on Windows. This page is very informative, but is certainly missing WSL 2 which is a very big hole.

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
